scripts/us_census/pep_srh/download.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+rm -rf ./download_files/
+rm -rf ./process_files/
+rm -rf ./output_files/
+
+mkdir -p ./download_files/1990_2000/county/
+mkdir -p ./download_files/2000_2010/county/
+mkdir -p ./download_files/2010_2020/county/
+
+mkdir -p ./process_files/agg/1990_2000/county/
+mkdir -p ./process_files/agg/2000_2010/county/
+mkdir -p ./process_files/agg/2010_2020/county/
+
+mkdir -p ./process_files/as_is/1990_2000/county/
+mkdir -p ./process_files/as_is/2000_2010/county/
+mkdir -p ./process_files/as_is/2010_2020/county/
+
+mkdir -p ./download_files/1980_1990/state/
+mkdir -p ./download_files/1990_2000/state/
+
+mkdir -p ./process_files/as_is/1980_1990/state/
+mkdir -p ./process_files/as_is/1990_2000/state/
+mkdir -p ./process_files/as_is/2000_2010/state/
+mkdir -p ./process_files/as_is/2010_2020/state/
+
+mkdir -p ./process_files/agg/1980_1990/state/
+mkdir -p ./process_files/agg/1990_2000/state/
+mkdir -p ./process_files/agg/2000_2010/state/
+mkdir -p ./process_files/agg/2010_2020/state/
+
+mkdir -p ./process_files/as_is/1980_1990/national/
+mkdir -p ./process_files/as_is/1990_2000/national/
+mkdir -p ./process_files/as_is/2000_2010/national/
+mkdir -p ./process_files/as_is/2010_2020/national/
+
+mkdir -p ./process_files/agg/1980_1990/national/
+mkdir -p ./process_files/agg/1990_2000/national/
+mkdir -p ./process_files/agg/2000_2010/national/
+mkdir -p ./process_files/agg/2010_2020/national/
+
+mkdir -p ./output_files/
+
+for file in `cat download_county_files_1990_2000.txt`
```

Comment:
safer to use `$( command ...)` instead of backticks to execute shell commands.
`$(cat download_county_files_1990_2000.txt)`

Reply:
changed to $() - command substitution.